roles/privacyidea/tasks/security.yml

```diff
@@ -33,22 +27,22 @@
     job: "[ -f /var/run/reboot-required ] && /usr/sbin/reboot"
 
 - name: Copy PrivacyIDEA Fail2Ban filter
-  copy:
-    src: privacyidea.conf
-    dest: /etc/fail2ban/filter.d/privacyidea.conf
+  template:
+    src: auth-fail2ban.conf.j2
+    dest: /etc/fail2ban/filter.d/auth-fail2ban.conf
   notify: restart fail2ban
 
 - name: Copy PrivacyIDEA Fail2Ban jail configuration
   template:
     src: privacyidea-jail.conf.j2
-    dest: /etc/fail2ban/jail.d/privacyidea.conf
+    dest: /etc/fail2ban/jail.d/privacyidea-jail.conf
   notify: restart fail2ban
 
 - name: copy ssh-jail for fail2ban
-  copy:
-    src: sshd.local
+  template:
+    src: sshd.local.j2
     dest: /etc/fail2ban/jail.d/sshd.local
-    owner: root
-    group: root
+    owner: www-data
+    group: www-data
     mode: 0744
```

Comment:
We set the owner and the permission here but not for privacyidea-jail.conf (above). Is there a reason for that?

Reply:
i do not think so, but i do not know. Since the fail2ban service is running as root it may be best to delete owner and group (so they belong to the root user and group) as it is the case for the privacyidea-jail.conf?